urllib3/poolmanager.py

```diff
@@ -65,6 +65,10 @@ def __init__(self, num_pools=10, headers=None, **connection_pool_kw):
         self.pools = RecentlyUsedContainer(num_pools,
                                            dispose_func=lambda p: p.close())
 
+        # Locally set the pool classes so other PoolManagers can override them.
+        if not hasattr(self, 'pool_classes_by_scheme'):
```

Comment:
Oh, actually, maybe. It depends how careful you want to be about the order in which people call `super()` and then do their own initialization.